config/eslint/eslintConfig.json

```diff
@@ -1,40 +1,42 @@
 {
   "root": true,
-  "extends": ["./config/eslint/eslintBaseConfig.json", "eslint-config-prettier"],
+  "extends": [
```

Comment:
Most of the ESLint changes in this PR are direct copy/paste from the ESLint library. Once that gets open sourced, I can remove all the copied code.